Autopilot/Inc/airspeed.hpp

```diff
@@ -0,0 +1,59 @@
+/**
+*   
+*   Airspeed sensor functions
+*
+*   This header is an outline of basic functions of the airspeed
+*   sensor. Generally, this sensor is a Pitot tube which uses a 
+*   pressure differential to determine airspeed. 
+*
+*   Airspeed calculation should be done and returned in meters 
+*   per second
+*
+*    Author: Annie LePage
+**/
+
+
+#ifndef AIRSPEED_HPP
+#define AIRSPEED_HPP
+
+#include <stdint.h>
+/*
+    Currently there is only one airspeed sensor used, if this is 
+    changed to a different sensor, part numbers and a selection
+    option should be added
+*/
+
+struct airspeedData_t 
+{
+    double airspeed;    // in m/s
+
+    float utcTime;      // 4 Bytes. Time in seconds since 00:00 
```

Comment:
There should be some sort of sensorStatus field (Lucy and I made it an integer) so we can report any error codes.

Also, there should be an isDataNew flag (probably a bool) so a caller can check whether data has been updated since the last time they called this function.

Reply:
I can do that